week-4-practise/level-2/exercise-1.js

```diff
@@ -35,7 +50,7 @@ bottle.fill();
 bottle.drink();
 bottle.drink();
 bottle.drink();
-if(!bottle.empty()){
-	console.log('bottles volume = ' + bottle.volume);
+if (!bottle.empty()) {
+  console.log("bottles volume = " + bottle.volume);
 }
-console.log("Above volume should be: 70");
\ No newline at end of file
+console.log("Above volume should be: 70");

```

Comment:
To test your program add these lines of code. The comment next to each `console.log` line represent what the output on the console should be:

```
console.log(bottle.volume) // 0

bottle.fill()
console.log(bottle.volume) //100

bottle.drink()
console.log(bottle.volume) //90

bottle.drink()
console.log(bottle.volume) // 80
console.log(bottle.empty()) // false


bottle.drink()
bottle.drink()
bottle.drink()
bottle.drink()
bottle.drink()
bottle.drink()
bottle.drink()
bottle.drink()
bottle.drink()
bottle.drink()
console.log(bottle.empty()) // true
console.log(bottle.volume) // 0
```

I purposely drank the bottle more than 10 times to make sure that you are not subtracting to negative values.

If you are unsure on how to achieve this, I would recommend to revise this page `https://github.com/CodeYourFuture/js-exercises/tree/master/week-4/D-methods`.